wheel/build_wheel_manylinux.sh

```diff
@@ -39,11 +39,12 @@ function audit_tlcpack_wheel() {
 
     cd "${TVM_PYTHON_DIR}" && \
       mkdir -p repared_wheel && \
-      auditwheel repair ${AUDITWHEEL_OPTS} dist/tlcpack*cp${python_version_str}*.whl
+      auditwheel repair ${AUDITWHEEL_OPTS} dist/*cp${python_version_str}*.whl
 }
 
 TVM_PYTHON_DIR="/workspace/tvm/python"
-PYTHON_VERSIONS=("3.6" "3.7" "3.8")
+PYTHON_VERSIONS_CPU=("3.6" "3.7" "3.8" "3.9" "3.10")
+PYTHON_VERSIONS_GPU=("3.6" "3.7" "3.8")
```

Comment:
```suggestion
PYTHON_VERSIONS_CPU=("3.7" "3.8" "3.9" "3.10")
PYTHON_VERSIONS_GPU=("3.7" "3.8")
```
since we don't support 3.6 in TVM do we need it here?

Reply:
It needs removing in all platforms. J thought about doing that in a separarte PR

Reply:
https://github.com/tlc-pack/tlcpack/issues/107 for agreement and future PR.